spec/decorator.rb

```diff
@@ -0,0 +1,24 @@
+require './nameable'
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Kindly make this require './nameable' something like the below 👍

1. require './spec/nameable'
2. require_relative './nameable'

I have given two ways you can load your file properly because you are not loading your file properly so kindly choose any of the above to make your app work as expected 👍

Reply:
Already done; thanks :pray: